docs/4.3/openssh_teleport.md

```diff
@@ -0,0 +1,311 @@
+in "recording mode", you have to tell the `ssh` client to use the jump host and
+enable the agent forwarding, otherwise a recording proxy will not be able to
+terminate the SSH connection to record it:
+
+``` bash
+# Note that agent forwarding is enabled twice: one from a client to a proxy
+# (mandatory if using a recording proxy), and then optionally from a proxy
+# to the end server if you want your agent running on the end server or not
+$ ssh -o "ForwardAgent yes" \
+    -o "ProxyCommand ssh -o 'ForwardAgent yes' -p 3023 %r@p.example.com -s proxy:%h:%p" \
+    user@host.example.com
+```
+
+!!! tip "Tip"
+
+    To avoid typing all this and use the usual `ssh user@host.example.com `, users
+    can update their ` ~/.ssh/config` file.
+
+**IMPORTANT**
+
+It's important to remember that SSH agent forwarding must be enabled on the
+client. Verify that a Teleport certificate is loaded into the agent after
+logging in:
+
+``` bsh
+# Login as Joe
+$ tsh login --proxy=proxy.example.com --user=joe
+# see if the certificate is present (look for "teleport:joe") at the end of the cert
+$ ssh-add -L
+```
+
+!!! warning "GNOME Keyring SSH Agent and GPG Agent"
+
+    It is well-known that Gnome Keyring SSH agent, used by many popular Linux
+    desktops like Ubuntu, and gpg-agent from GnuPG do not support SSH
+    certificates. We recommend using the `ssh-agent` from OpenSSH.
+    Alternatively, you can disable SSH agent integration entirely using
+    `--no-use-local-ssh-agent` flag or `TELEPORT_USE_LOCAL_SSH_AGENT=false`
+    environment variable with `tsh`.
+
+
+## Using OpenSSH Client
+
+It is possible to use the OpenSSH client `ssh` to connect to nodes within a
+Teleport cluster. Teleport supports SSH subsystems and includes a `proxy`
+subsystem that can be used like `netcat` is with `ProxyCommand` to connect
+through a jump host.
+
+On your client machine, you need to import these keys. It will allow your
+OpenSSH client to verify that host's certificates are signed by the trusted CA
+key:
+
+``` yaml
+$ cat teleport_user_ca.pub >> ~/.ssh/known_hosts
+```
+
+If you have multiple Teleport clusters, you have to export and set up these
+certificate authorities for each cluster individually.
+
+!!! tip "OpenSSH and Trusted Clusters"
+
+    If you use [recording proxy mode](architecture/teleport_proxy.md) and [trusted
+    clusters](trusted-clusters.md), you need to set up certificate authority from
+    the _root_ cluster to match **all** nodes, even those that belong to _leaf_
+    clusters. For example, if your node naming scheme is `*.root.example.com`,
+    `*.leaf1.example.com`, `*.leaf2.example.com`, then the
+    `@certificate-authority` entry should match `*.example.com` and use the CA
+    from the root auth server only.
+
+Make sure you are running OpenSSH's `ssh-agent` , and have logged in to the
+Teleport proxy:
+
+``` bash
+$ eval `ssh-agent`
+$ tsh --proxy=work.example.com login
```

Comment:
```suggestion
$ tsh --proxy=root.example.com login
```

We should pick a naming scheme and use it throughout the example to make it easier to follow. I think `root.example.com` and `leaf1.example.com` are fine.